vue-app/src/components/Cart.vue

```diff
@@ -49,14 +49,12 @@
           <div>Choose some projects that you want to contribute to...</div>
           <router-link to="/projects" class="btn-secondary mobile mt1">See projects</router-link>
         </div>
-        <div v-else-if="$store.getters.canUserReallocate && !isCartEmpty">
+        <div v-else-if="$store.getters.canUserReallocate && !isCartEmpty && !$store.getters.hasReallocationPhaseEnded">
```

Comment:
This addition is redundant - if `$store.getters.canUserReallocate` is `true`, then `!$store.getters.hasReallocationPhaseEnded` is also `true`.